stdlib/typing.pyi

```diff
@@ -820,7 +820,13 @@ class ForwardRef:
     else:
         def __init__(self, arg: str, is_argument: bool = ...) -> None: ...
 
-    def _evaluate(self, globalns: dict[str, Any] | None, localns: dict[str, Any] | None) -> Any | None: ...
+    if sys.version_info >= (3, 11):
```

Comment:
python/cpython#21553 was backported to 3.9

```suggestion
    if sys.version_info >= (3, 9):
```

Reply:
Done, thanks!